test/cell_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+require './lib/cell'
+
+class CellTest < Minitest::Test
+
+  def setup
+    skip
+    @cell = Cell.new("B4")
+  end
+
+  def test_it_exists
+    skip
+    assert_instance_of Cell, @cell
+  end
+
+  def test_it_has_coordinates
+    skip
+    assert_equal "B4", @cell.coordinate
+  end
+
+  def test_it_does_not_have_a_ship
+    skip
+    assert_equal nil, @cell.ship
+  end
+
+  def test_it_is_empty
+    skip
+    assert @cell.empty?
+  end
+
+  def test_it_has_a_new_ship
+    skip
+    assert_equal @cruiser, @cell.place_ship
```

Comment:
@amymoller 

So here you'd want to add that line that does * something * but has no return value in the interaction pattern: `@cell.place_ship(cruiser)`. 

You'll also want to define `cruiser` first as well, before calling that previous line. `cruiser = Ship.new("Cruiser", 3)`

Then the thing you're testing is the line `cell.ship`, which should return an instance of `Ship` class (according to the interaction pattern).

At this point, you would want to test for that next piece you wrote in line #40, but inside **this** test, because it relies on all the information above.
`refute @cell.empty?`